wandb/sdk/artifacts/artifact.py

```diff
@@ -103,13 +103,11 @@ class Artifact:
         type: The artifact's type. Use the type of an artifact to both organize
             and differentiate artifacts. You can use any string that contains letters,
             numbers, underscores, hyphens, and dots. Common types include `dataset` or `model`.
-            Include `model` within your type string if you want to link the artifact
-            to the W&B Model Registry.
-        description: A description of the artifact. For Model or Dataset Artifacts,
-            add documentation for your standardized team model or dataset card. View
+            For W&B Registry users, ensure that the artifact type you specify is
+            permitted by the registry you want to link the artifact to.
+        description: A description of the artifact. View
             an artifact's description programmatically with the `Artifact.description`
-            attribute or programmatically with the W&B App UI. W&B renders the
-            description as markdown in the W&B App.
```

Comment:
I still think we should keep the note on markdown - this informs to the user what they should pass in when setting a description. .